st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -84,17 +83,10 @@ def _set_up_logger(self):
         Set up a logger which logs all the messages with level DEBUG
         and above to stderr.
         """
+        logging.config.fileConfig('/etc/st2actions/logging.conf')
```

Comment:
This path can't be hard-coded and needs to be configurable.

Please see the existing code on how we handle those paths - those values are provided in the config and can be overridden by the user.